config/environments/development.rb

```diff
@@ -40,17 +40,10 @@
   # config.action_view.raise_on_missing_translations = true
 
   # ActionMailer Config
-  config.action_mailer.smtp_settings = {
-      address: 'smtp.sendgrid.net',
-      port: '587',
-      authentication: :plain,
-      user_name: ENV['SENDGRID_USERNAME'],
-      password: ENV['SENDGRID_PASSWORD'],
-      domain: 'heroku.com',
-      enable_starttls_auto: true
-  }
-  config.action_mailer.default_url_options = { :host => 'localhost:3000' }
-  config.action_mailer.delivery_method = :letter_opener
+  config.action_mailer.smtp_settings = { address: "localhost",
+                                         port: 1025 }
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.